### PR TITLE
feat(FX-4304): add activity panel tabs

### PR DESCRIPTION
**Pull request by dimatretyak**
#### iOS
https://user-images.githubusercontent.com/3513494/193029358-73d34ca2-49b0-45e3-9396-59eeab68418f.mp4

#### Android
https://user-images.githubusercontent.com/3513494/193030466-7db96a63-bb7d-4052-8a4c-604973b67572.mp4

This PR resolves [FX-4304]

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add tabs for activity screen - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4304]: https://artsyproduct.atlassian.net/browse/FX-4304?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Activity/ActivityTabSubheader.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const entityByType: Record<NotificationType, SubheaderEntity> = {
+  all: {
+    title: "All",
+  },
+  alerts: {
+    title: "Alerts",
+    buttonLabel: "Manage alerts",
+  },
+}
```
Comment (dimatretyak) [inline src/app/Scenes/Activity/ActivityTabSubheader.tsx:48]: **question**: or is it better to use [this approach](https://github.com/artsy/eigen/pull/7412/commits/0ab0162075885ec631d66e2de4c498b8273731fa#diff-060f99a1cbc0156e48921f191ad5e8e6304901f42047f45fe4045f5dd47e22c7R49-R56)?

File: src/app/Scenes/Activity/Activity.tsx [hunk 2]
```diff
@@ -1,37 +1,82 @@
...
+  ]
+
+  return (
+    <Flex flex={1}>
+      <StickyTabPage
+        tabs={tabs}
+        staticHeaderContent={
+          <FancyModalHeader onLeftButtonPress={goBack} hideBottomDivider>
+            Activity
+          </FancyModalHeader>
```
Comment (gkartalis) [inline src/app/Scenes/Activity/Activity.tsx:54]: **question:** is there a reason why we are using fancymodalheader hear and not any other header? 🤔 asking out of curiosity

File: src/app/Scenes/Activity/ActivityList.tsx [hunk 3]
```diff
@@ -1,23 +1,42 @@
...
+    key: notification.internalID,
+    content: <ActivityItem item={notification} />,
+  }))
+  const sections: StickyTabSection[] = [
+    {
+      key: SUBHEADER_SECTION_KEY,
+      content: <ActivityTabSubheader type={type} />,
+    },
+    ...notificationSections,
+  ]
```
Comment (gkartalis) [inline src/app/Scenes/Activity/ActivityList.tsx:39]: 👏

File: src/app/Scenes/Activity/ActivityTabSubheader.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Scenes/Activity/ActivityTabSubheader.tsx:48]: this one looks cleaner to me 🤔

**Review by gkartalis — APPROVED** [review]

File: src/app/Scenes/Activity/ActivityTabSubheader.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/app/Scenes/Activity/ActivityTabSubheader.tsx:48]: do you mean current approach?

Comment (gkartalis) [reply src/app/Scenes/Activity/ActivityTabSubheader.tsx:48]: yep

File: src/app/Scenes/Activity/Activity.tsx (hunk 2 above)
Comment (dimatretyak) [reply src/app/Scenes/Activity/Activity.tsx:54]: as far as I remember, only the `FancyModalHeader` consists of a back button and a header title in the center (what we have by design). Other components either include only the back button, or differ in design from what we need

Comment (dimatretyak) [reply src/app/Scenes/Activity/Activity.tsx:54]: and we can also use this component for `StickyTabPage` without any problems (just pass it to `staticHeaderContent` prop)

**Pull request merged**